#goal definition
float32 initial_position
float32 final_position
float32 max_angular_velocity
---
#result definition
float32 reached_position
---
#feedback
float32 current_position
